### PR TITLE
Desktop version review

### PR DESCRIPTION
**Pull request by akezeth**
Added structure and style for the desktop version of the Figma design template 2.
Used media query with min-width = 768px.
Used both Grid and Flexbox for styling the structure of the page.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -60,43 +79,32 @@ <h3 class="job-title">
                 <h2 class="recentWorks">
```
Comment (topeogunleye) [inline index.html:79]: - [ ] As written in the project requirements. I kindly suggest that you ensure that when a project is hovered on with the mouse:
-  the description and dark opacity disappear
-  and a See Project button appears to the bottom of the image and takes the whole image width.
-   Your `See Project` button should link to the Live Demo of the project or to the GitHub repository.
-   The project text should take up no more than 3 lines of text. Your project text currently takes up 4 lines of text.

| Your Design  |Template Design |
| ------------- | ------------- |
| image | image |

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @akezeth, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Nothing to mention.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by akezeth** [comment]
Added hover state property for the work div elements.
Added link for See Project button.

File: index.html [hunk 2]
```diff
@@ -60,131 +79,146 @@ <h3 class="job-title">
...
```
Comment (Zuhaib042) [inline index.html:79]: - [ ] What the previous reviewer meant here was the following:- 
  -  First, without hovering the mouse on the card, the `see button` should not be there. 
  -  Then, when hovering the mouse on card, the description and dark opacity should disappear and `see button` shows up. you can achieve this by using `:hover` pseudoclass, and not with the click.
Right now, the hover effect is not working on these cards. I had to click, and even on clicking, the desired effect is not there.
 your design now.

https://user-images.githubusercontent.com/96683843/229208198-c4b7b2f1-c3ff-4151-b7af-44e044e36299.mov

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @akezeth , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
Followed Github flow ✔️
Professional README file. ✔️
PR has a good title and summary ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by topeogunleye** [comment]
> Added hover state property for the work div elements. Added link for See Project button.



> Added hover state property for the work div elements. Added link for See Project button.

@akezeth Here's of video demo of the Expected App behavior:

https://www.loom.com/share/66329832a272445b989fa32dac2e5796

**Comment by akezeth** [comment]
Modified hover state property for the work div elements.

**Review by TadesseDev — APPROVED** [review]
### Hi @akezeth 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**